angular/defs.bzl

```diff
@@ -134,7 +134,10 @@ def ng_app(name, project_name = None, deps = [], test_deps = [], **kwargs):
         name = "test",
         chdir = native.package_name(),
         args = ["%s:test" % project_name],
+        include_declarations = True,
         data = srcs + test_srcs + deps + test_deps + TEST_DEPS + TEST_CONFIG + COMMON_CONFIG,
+        # TODO: fix in sandbox
```

Comment:
What fix is required? Is there a link we can put here?

Reply:
I don't know yet. Needs more investigation.

Reply:
Couldn't figure it out quickly. Will need to do some regression testing on this one to find out what commit broke it.